tests/src/OneLogin/saml2_tests/authn_request_test.py

```diff
@@ -255,7 +255,7 @@ def testCreateEncSAMLRequest(self):
         inflated = decompress(decoded, -15)
 
         self.assertRegexpMatches(inflated, '^<samlp:AuthnRequest')
-        self.assertRegexpMatches(inflated, 'AssertionConsumerServiceURL="http://stuff.com/endpoints/endpoints/acs.php">')
+        self.assertRegexpMatches(inflated, 'AssertionConsumerServiceURL="http://stuff.com/endpoints/endpoints/acs.php"(\s)*>')
```

Comment:
why this change is required?

Can you add more unit test to cover that new attributeConsumingService feature and adapt old tests?